### PR TITLE
Add missing color for ABORTED outcome in ConsoleSummary.

### PR DESCRIPTION
**Pull request by DanLipsitt**

### CODE REVIEW

**Comment by DanLipsitt** [comment]
Not needed in current master (170dd72).

**Comment by DanLipsitt** [comment]
Not sure why I thought this was fixed elsewhere, but it isn't. This PR is still needed.

**Comment by DanLipsitt** [comment]
PTAL

File: test/output/callbacks/callbacks_test.py [hunk 1]
```diff
@@ -58,3 +59,12 @@ def test_testrun(self, user_mock):
...
     mfg_inspector.OutputToTestRunProto(testrun_output)(record)
+
+
+class TestConsoleSummary(test.TestCase):
+
+    def test_outcome_colors(self):
+        """Ensure there is an output color for each outcome."""
+        instance = console_summary.ConsoleSummary()
+        for outcome in htf.test_record.Outcome:
+            self.assertIsNotNone(instance.color_table[outcome])
```
Comment (fahhem) [inline test/output/callbacks/callbacks_test.py]: self.assertIn(outcome, instance.color_table)

Comment (DanLipsitt) [inline test/output/callbacks/callbacks_test.py]: Oops. Good call.

**Comment by DanLipsitt** [comment]
Fixed. PTAL.

**Pull request merged**